### PR TITLE
OCLOMRS-488: Fix console error when application runs.

### PR DESCRIPTION
**Pull request by judeinno**
# JIRA TICKET NAME:
[Fix console error when application runs.](https://issues.openmrs.org/browse/OCLOMRS-488)

# Summary:
When the application runs there is a console error that states 

`Uncaught SyntaxError: Unexpected token <`

I should be removed.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,7 +12,6 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="msapplication-config" content="%PUBLIC_URL%/images/favicons/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
-    <script src="./env-config.js"></script>
```
Comment (dkayiwa) [inline public/index.html]: Does this mean that you no longer need env-config.js?

Comment (judeinno) [reply public/index.html]: Yes, It's the source of the error because the file does not exist.

Comment (judeinno) [reply public/index.html]: I have also reached out to Rafal on [Talk](https://talk.openmrs.org/t/developing-the-ocl-for-openmrs-application/17771/806?u=judeatu) for clarification.

**Comment by judeinno** [comment]
Rafal provided clarity on [Talk](https://talk.openmrs.org/t/developing-the-ocl-for-openmrs-application/17771/807?u=judeatu) and he updated then code [here](https://github.com/openmrs/openmrs-ocl-client/commit/fbd871d18a25ff31dd233e26b3200407f0a8f884) where he requires it.

**Pull request closed without merging**